cetmix_tower_server/views/cx_tower_command_view.xml

```diff
@@ -191,6 +191,12 @@ COMMAND_RESULT = {"exit_code": x, "message": "This will be logged as an error me
                     name="file_from_template"
                     domain="[('action', '=', 'file_using_template')]"
                 />
+
+                <filter
```

Comment:
Put this filter right after the "shell command" and rename the filters:

- SSH Command
- Python Code
- File from Template